.storybook/package.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
question: is this intentionally blank?

Reply:
Yes, unfortunately it is a work around for a quirk with storybook and webpack 5 where you need to trick it into thinking it's a module. I have since updated a bunch more deps, so I need to double check if the work around is still required